filters/f_output_chain.c

```diff
@@ -357,6 +357,15 @@ static double get_display_fps(struct mp_stream_info *i)
     return res;
 }
 
+static int *get_display_res(struct mp_stream_info *i)
+{
+    struct chain *p = i->priv;
+    static int res[2] = {0};
```

Comment:
what no
static locals are to be avoided at all costs

Reply:
to be clear: what I meant from the beginning was that the signature should be `static void get_display_res(struct mp_stream_info *i, int *dst)`

Reply:
oh